src/components/product-menu/product-menu.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import PopoverTrigger from '@mapbox/react-popover-trigger';
+import Icon from '@mapbox/react-icon';
+import { ProductMenuDropdown } from '../product-menu-dropdown/product-menu-dropdown';
+// You could import this list from, say, a data selector if you like
+// On Mapbox docs pages, categories data is imported from a central location
+// to keep menus in sync across the site.
+import { ProductMenuItems } from '../../data/product-menu-items';
+
+const popoverProps = {
+  placement: 'bottom',
+  themePopover:
+    'round shadow-darken25 viewport-almost-but-not-always scroll-auto scroll-styled'
+};
+
+class ProductMenu extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = { clicked: false };
+
+    this.onPopoverOpen = this.onPopoverOpen.bind(this);
+    this.onPopoverClose = this.onPopoverClose.bind(this);
+  }
+
+  renderMenu() {
+    return <ProductMenuDropdown categories={ProductMenuItems} />;
```

Comment:
Could we generalize this component a bit more by rendering `ProductMenu`'s children instead of `ProductMenuDropdown` specifically?

Reply:
I'm not sure it needs to be generalized, really, unless we decide at some point that we want to repeat the button style with different content. For now, these components are coupled immediately with the product names themselves, and I think that's ok.

Reply:
@davidtheclark 🙈 you're so fast! whelp, I rearranged the furniture a bit and I think I like it better this way. I don't know that we'll have an immediate use case for this style of button with other stuff inside of it, but I can see it being useful in the future. 

Would you and @colleenmcginnis mind taking a peek at the new arrangement and let me know if you think it's worse? 

![image](https://user-images.githubusercontent.com/2365503/42717671-7f40f61c-86b7-11e8-92fe-380a6ec830a8.png)

Reply:
👍